src/structures/Message.js

```diff
@@ -473,9 +486,15 @@ class Message extends Base {
    *   .then(interaction => console.log(`${interaction.customID} was clicked!`))
    *   .catch(console.error);
    */
-  awaitMessageComponentInteraction(options = {}) {
+  awaitMessageComponent(options = {}) {
+    const _options = {
+      ...options,
+      interactionType: InteractionTypes.MESSAGE_COMPONENT,
+      max: 1,
+      message: this,
+    };
```

Comment:
Because the other properties are always overridden in `createMessageComponentCollector`, we don't need to duplicate them again.
```suggestion
    const _options = { ...options, max: 1 };
```

You can also inline it like it was before.